contribs/gmf/src/services/permalink.js

```diff
@@ -146,6 +146,10 @@ gmf.Permalink = function($timeout, ngeoBackgroundLayerMgr, ngeoDebounce,
    */
   this.ngeoStateManager_ = ngeoStateManager;
 
+  if (gmfPermalinkOptions.useLocalStorage === false) {
+    this.ngeoStateManager_.localStorage = null;
```

Comment:
Or even `this.ngeoStateManager_.localStorage.isAvailable = ol.functions.FALSE;` ?